django_postal_codes/models.py

```diff
@@ -8,6 +8,7 @@
 from shapely import wkt
 from django.contrib.gis.db import models
 from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy
```

Comment:
can we do it the other way around?
```python
from django.utils.translation import gettext_lazy as _
from django.utils.translation import gettext
```
`gettext` with only be used in `__str__` methods, while the lazy version is used often. As such is better to use the lazy version as _ because it is less verbose